### ISSUE TITLE
Changes of amplitude in disabled sound channel produce cracking.

### ISSUE THREAD

**Issue by AntonyDTA** [simonowen/simcoupe#23]
In SimCoupe1.0 and SimCoupe1.1 (20150409) it's OK, but in (20200117) and newer it's bad.

**Comment by simonowen** [comment]
Recent builds of SimCoupe are using an updated SAASound library, which is bundled as SAASound.dll. I'll probably need @stripwax to suggest what might have changed to cause the problem. If it's not resolved by the next build I'll switch back to the older internal SAASound code.

Do you have a sample program or disk image that demonstrates the issue?

**Comment by stripwax** [comment]
Yep, happy to help if you can show me how to reproduce it.

**Comment by stefandrissen** [comment]
From Anton's facebook post:

````
10 FOR a=0 to 31: SOUND a,0: NEXT a
20 SOUND &1C,1
30 LET r=0: REM 0 to 5
40 SOUND r,&FF;r,0: GO TO 40
````

**Comment by stripwax** [comment]
Many thanks, I'll take a look

**Comment by stripwax** [comment]
Confirmed reproducible with 3.3.x version of SAASound.dll and not reproducible with 3.2.x version.
Tracking here: https://github.com/stripwax/SAASound/issues/16

**Comment by stripwax** [comment]
Fix deployed as SAASound 3.4 https://github.com/stripwax/SAASound/releases/tag/3.4

**Comment by simonowen** [comment]
Thanks Dave!

I've created an [updated release](https://github.com/simonowen/simcoupe/releases/tag/20200711) using the same SimCoupe source code but with the new SAASound.dll v3.4.0. The zip and installer files are dated today so you can tell them apart.
